sub/sd_ass.c

```diff
@@ -176,6 +177,9 @@ static int init(struct sd *sd)
             return -1;
         extradata = lavc_conv_get_extradata(ctx->converter);
         extradata_size = extradata ? strlen(extradata) : 0;
+
+        if (strcmp(sd->sh->codec, "eia_608") == 0)
+            ctx->duration_unknown = 1;
```

Comment:
The lavc_conv() function should probably return whether durations can be unknown.

Another possibility is that sd_ass.c checks the event list after calling ass_process_data(), but since libavcodec uses a relatively arbitrary timestamp for "infinite", that'd probably be less robust.